core/src/main/res/raw/ayp_youtube_player.html

```diff
@@ -75,6 +75,9 @@
                     
                     startSendCurrentTimeInterval();
                     sendVideoData(player);
+
+                    sendCurrentVideoIdFromPlaylist(player);
```

Comment:
i wonder if a better place for this would be `YT.PlayerState.BUFFERING`. What do you think?

Reply:
Hmm, both states (BUFFERING and PLAYING) have the disadvantage that they are are potentially entered multiple times while sending the same video id each time (for example if the user uses the seekbar to jump ahead). Although I assume that the BUFFERING state is reached first, which probably makes it the better place.

CUED does not work, since it seems to only be reached when a video (or playlist) is cued directly via a player function (`cueVideoById` or `cuePlaylist`) and not by interacting with the embedded playlist.

I can change the location to BUFFERING unless you have you an idea on how to prevent sending redundant video ids.

Reply:
> Hmm, both states (BUFFERING and PLAYING) have the disadvantage that they are are potentially entered multiple times while sending the same video id each time (for example if the user uses the seekbar to jump ahead). Although I assume that the BUFFERING state is reached first, which probably makes it the better place.

I didn't consider this. I'd say this has to be solved before we can merge this pull request. We don't want to send the video id each time the video is BUFFERING or PLAYING. As this is not the contract for the listener.

Reply:
I agree. I just looked into it again and think that the UNSTARTED event is fine. The doc says "When the player first loads a video, it will broadcast an unstarted (-1) event." I must have missed this before. I just tested it with the sample app and it works well: we only get a videoid once per video. Should I commit this change?

Reply:
Great news, thanks for checking! I will take a final look over the weekend :)